src/rules/utils.ts

```diff
@@ -652,6 +652,16 @@ export const isDescribe = (
   );
 };
 
+export const getNodeTitle = (node: TSESTree.CallExpression): string | null => {
```

Comment:
~Since this is only used by `valid-title`, I think it'd be best to have it in that rule for now, just to keep things in scope.~

~We can always move it into utils if it turns out useful in other rules :)~

After seeing it's usage in `valid-title`, I think it would make sense to just in-line this method for now, as it's doing a pretty standard node guard check :)

Reply:
I saw similar checks are made in rules like lower-case, no-identical-title etc. Hence I added this method thinking I would do necessary changes in other rules to use this method. 

If you feel this should be an inline thing will move to valid-title.

Reply:
I think for now I'd prefer it to be inlined, b/c for the same of avoiding extra functions & export, but thanks for raising this - I'll give the code base a sweep when I have a chance, and see if there are any duplications we could refactor into a utility function :)